components/organisms/Layout.js

```diff
@@ -0,0 +1,98 @@
+import PropTypes from "prop-types";
```

Comment:
text not collapsing properly

![Screen Shot 2021-04-01 at 10 32 41 AM](https://user-images.githubusercontent.com/26819992/113311440-5238a500-92d7-11eb-967f-73d69cb86af1.png)